lib/features/mfa/screens/mfa_screen.dart

```diff
@@ -1,40 +1,45 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:gruene_app/app/widgets/app_bar.dart';
 import 'package:gruene_app/features/mfa/bloc/mfa_bloc.dart';
 import 'package:gruene_app/features/mfa/bloc/mfa_event.dart';
 import 'package:gruene_app/features/mfa/bloc/mfa_state.dart';
 import 'package:gruene_app/features/mfa/widgets/intro_view.dart';
 import 'package:gruene_app/features/mfa/widgets/ready_view.dart';
 import 'package:gruene_app/features/mfa/widgets/verify_view.dart';
+import 'package:gruene_app/i18n/translations.g.dart';
 
 class MfaScreen extends StatelessWidget {
   const MfaScreen({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return BlocConsumer<MfaBloc, MfaState>(
-      listener: (context, state) {
-        if (state.error != null) {
-          ScaffoldMessenger.of(context).showSnackBar(
-            SnackBar(
-              content: Text(state.error!),
-            ),
-          );
-        }
-      },
-      builder: (context, state) {
-        switch (state.status) {
-          case MfaStatus.setup:
-            return const IntroView();
-          case MfaStatus.ready:
-            return const ReadyView();
-          case MfaStatus.verify:
-            return const VerifyView();
-          case MfaStatus.init:
-          default:
-            return const _InitView();
-        }
-      },
+    return Scaffold(
```

Comment:
Most of these changes are only wrapping the screens in a scaffold to allow for individually setting an appBar.